product_abc_classification/README.rst

```diff
@@ -0,0 +1,108 @@
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fproduct--attribute-lightgray.png?logo=github
+    :target: https://github.com/OCA/product-attribute/tree/13.0/product_abc_classification
+    :alt: OCA/product-attribute
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/product-attribute-13-0/product-attribute-13-0-product_abc_classification
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/135/13.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5|
+
+This modules includes the ABC analysis (or ABC classification), which is
+used by inventory management teams to help identify the most important
+products in their portfolio and ensure they prioritize managing them above
+those less valuable.
+
+Managers will create a profile with several levels (percentages) and then the
+profiled products will automatically get a corresponding level using the
+ABC classification.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to Sales or Inventory menu, then to Configuration/Products/ABC Classification Profile
+and create a profile with levels. If the classification type is "Percentage" you have to
+know that the sum of all levels in the profile should sum 100 and all the levels should
+be different.
+
+#. Later you should go to product categories or product variants, and assign them a profile.
+Then the cron classification will proceed to assign to these products one of the profile's levels.
+
+NOTE: If you profile (or unprofile) a product category, then all its
+child categories and products will be profiled (or unprofiled).
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/product-attribute/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/product-attribute/issues/new?body=module:%20product_abc_classification%0Aversion:%2013.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* ForgeFlow
+
+Contributors
+~~~~~~~~~~~~
+
+* Miquel Raïch <miquel.raich@forgeflow.com>
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+  * Carlos Roca
+  * Stefan Ungureanu
```

Comment:
You shouldn't edit this manually, but run the OCA tool to generate the README. If not able to run, simply don't touch and let the bot to modify it on merge.